openwisp_radius/tests/tests.py

```diff
@@ -618,7 +618,7 @@ class TestOgranizationRadiusSettings(ApiTokenMixin, BaseTestCase):
     user_model = User
 
     def setUp(self):
-        self.org = self._create_org()
+        self.org = self._create_org(name='test', slug='test')
```

Comment:
Why do we need to pass these arguements now ? :confused:

Reply:
The function returns the default organisation if no arguments are passed. In the test, new `OrganizationRadiusSettings` are created for this organisation. As now the settings will be already created, it would cause errors.